step-plans/step-plans-base-artefacts/pom.xml

```diff
@@ -100,6 +100,18 @@
 			<version>${slf4j.version}</version>
 		</dependency>
 
+		<dependency>
+			<groupId>io.kubernetes</groupId>

```

Comment:
@rubij see comment on the TokenAutoscalingConfiguration class, should be removed.

Reply:
Done